docs/static/resources/openapi.json

```diff
@@ -5454,25 +5454,6 @@
         },
         "type": "object"
       },
-      "ExplorePermalinkPostSchema": {
```

Comment:
should you just rename this to `ExplorePermalinkPostSchema`? 
maybe just use `https://github.com/preset-io/superset/blob/master/superset/cli/update.py#L71` to update?

Reply:
Nice, it looks like the cli command hasn't been run in a while. I've committed the output, if we think it's worth cleaning the whole file up in the process.